### DISCUSSION TITLE
Question about usage of `send(__method__)`

### DISCUSSION THREAD

**Discussion by omkarmoghe** [AaronLasseigne/active_interaction#570] [Q&A] [answered]
Had a maybe basic Ruby question on the usage of `send(__method__)` in the filter code, specifically in the `cast` method. 

I was wondering why the code uses `send(__method__)` at all instead of just calling `cast` directly? Is there a difference between the two that I'm missing?

**Comment by AaronLasseigne** [comment, marked as answer]
It's the same thing. The only real difference is that if I rename the `cast` method I don't have to worry about fixing the name inside of the method.

**Reply by omkarmoghe** [reply]
Makes sense, thanks!
